lld/COFF/Chunks.h

```diff
@@ -835,18 +835,30 @@ class ECExportThunkChunk : public NonSectionCodeChunk {
   Defined *target;
 };
 
+// ARM64X relocation value, potentially relative to a symbol.
+class Arm64XRelocVal {
+public:
+  Arm64XRelocVal(int64_t value = 0) : value(value) {}
```

Comment:
This constructor takes an `int64_t`, while the `value` field actually is an `uint64_t`. Is this intended?

Reply:
I changed argument type to `uint64_t`. Negative values are possible to delta values, but the cast may just as well happen on the call. Thanks!